spec/support/page_objects/publish/courses/new_modern_languages_page.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module PageObjects
+  module Publish
+    module Courses
+      class NewModernLanguages < PageObjects::Base
+        set_url "/publish/organisations/{provider_code}/{recruitment_cycle_year}/courses/modern-languages/new{?query*}"
+
+        element :languages_fields, '[data-qa="course__languages"]'
+        element :title, '[data-qa="page-heading"]'
+
+        def language_checkbox(name)
```

Comment:
Could we place the methods last before the DSLs

Reply:
done in https://github.com/DFE-Digital/teacher-training-api/pull/2525/commits/77827debc4ed2be38058f60183926ff5264ee34a